packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -245,7 +245,7 @@ class EditorPage {
 			blockAccessibilityLabel
 		);
 		const size = await this.driver.getWindowSize();
-		const height = size.height - 5;
+		const height = size.height - 50;
```

Comment:
For future reference, what's the typical safe area height we assume at this point, and what's the block picker height? Trying to understand if 50 is still very close to the safe area or not.

Reply:
While at it, I think it makes sense to add a line comment about the hardcoded value, explaining a bit how we decided on the specific value and what's its role.

Reply:
Safe are is 34 in portrait and 21 on landscape. The picker is 464 in iPhone 11 (the one used in e2e tests).